common/lib/xmodule/xmodule/js/src/video/09_video_caption.js

```diff
@@ -1137,6 +1139,8 @@
                     .removeClass('is-active')
                     .find('.control-text')
                         .text(gettext('Turn on closed captioning'));
+
```

Comment:
Just curious -- what is the pattern for placement of the event trigger within the method.   Is it supposed to be triggered right after the action it's reporting, or right before?